task/build-paketo-builder-oci-ta/0.2/build-paketo-builder-oci-ta.yaml

```diff
@@ -390,8 +390,8 @@ spec:
 
         sboms_to_merge=(syft:sbom-source.json syft:sbom-image.json)
 
-        if [ -f "sbom-cachi2.json" ]; then
-          sboms_to_merge+=(cachi2:sbom-cachi2.json)
+        if [ -f "sbom-prefetch.json" ]; then
+          sboms_to_merge+=(hermeto:sbom-prefetch.json)
```

Comment:
I'm struggling to figure out how this filename changed. It's the only reference to sbom-prefetch.json in this file, unlike the buildah task which copies the Hermeto SBOM to sbom-prefetch.json.

I suspect the handling for the Hermeto SBOM never worked in the first place. No need to fix it here if that's the case. @cmoulliard FYI